<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>20</y>
     <width>323</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>日历选择</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>选择的日期：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="editCalendar"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QCalendarWidget" name="calendarWidget"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>258</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnClose">
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>258</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>531</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>日期时间</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QPushButton" name="btnGetTime">
         <property name="text">
          <string>读取当前日期时间</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="LabDateTime">
         <property name="text">
          <string>字符串显示</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>时 间</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTimeEdit" name="timeEdit">
         <property name="dateTime">
          <datetime>
           <hour>15</hour>
           <minute>30</minute>
           <second>55</second>
           <year>2000</year>
           <month>1</month>
           <day>1</day>
          </datetime>
         </property>
         <property name="displayFormat">
          <string>HH:mm:ss</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="editTime"/>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="btnSetTime">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>设置时间</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>日 期</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDateEdit" name="dateEdit">
         <property name="dateTime">
          <datetime>
           <hour>0</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2016</year>
           <month>11</month>
           <day>21</day>
          </datetime>
         </property>
         <property name="displayFormat">
          <string>yyyy年M月d日</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="editDate">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="btnSetDate">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>设置日期</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>日期时间</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="editDateTime">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="btnSetDateTime">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>设置日期时间</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDateTimeEdit" name="dateTimeEdit">
         <property name="dateTime">
          <datetime>
           <hour>0</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2018</year>
           <month>10</month>
           <day>9</day>
          </datetime>
         </property>
         <property name="displayFormat">
          <string>yyyy-MM-dd HH:mm:ss</string>
         </property>
         <property name="calendarPopup">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>定时器</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0" colspan="2">
          <widget class="QPushButton" name="btnStart">
           <property name="text">
            <string>开始</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QSpinBox" name="spinBoxIntv">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="suffix">
            <string> ms</string>
           </property>
           <property name="maximum">
            <number>999999</number>
           </property>
           <property name="value">
            <number>1000</number>
           </property>
           <property name="displayIntegerBase">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QPushButton" name="btnSetIntv">
           <property name="text">
            <string>设置周期</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>定时周期</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="LabElapsTime">
           <property name="text">
            <string>流逝时间</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" colspan="2">
          <widget class="QPushButton" name="btnStop">
           <property name="maximumSize">
            <size>
             <width>93</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>停止</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLCDNumber" name="LCDHour">
           <property name="digitCount">
            <number>2</number>
           </property>
           <property name="value" stdset="0">
            <double>10.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="LCDMin">
           <property name="digitCount">
            <number>2</number>
           </property>
           <property name="value" stdset="0">
            <double>26.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="LCDSec">
           <property name="digitCount">
            <number>2</number>
           </property>
           <property name="value" stdset="0">
            <double>35.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>711</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
